WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -164,6 +169,9 @@ private extension OrderDetailsViewController {
         editButton.accessibilityIdentifier = "order-details-edit-button"
         editButton.isEnabled = viewModel.editButtonIsEnabled
         navigationItem.rightBarButtonItems = [editButton] + orderNavigationRightBarButtonItems()
+
+        // Disables large title.
```

Comment:
I don't think this comment adds any value, the quick help can easily explain this if someone doesn't know what it does.
```suggestion
```